src/main/java/me/zeroeightsix/kami/gui/hudgui/elements/combat/Armor.kt

```diff
@@ -55,9 +55,13 @@ object Armor : HudElement(
         safeAsyncListener<TickEvent.ClientTickEvent> { event ->
             if (event.phase != TickEvent.Phase.END) return@safeAsyncListener
 
-            for ((index, item) in armorItems.withIndex()) {
-                armorCounts[index] = InventoryUtils.countItemAll(item)
-            }
+            armorCounts[0] = InventoryUtils.countItemAll(Items.DIAMOND_HELMET)
+            armorCounts[1] = InventoryUtils.countItemAll(
+                if (countElytras.value && mc.player.inventory.getStackInSlot(38).item == Items.ELYTRA) Items.ELYTRA
```

Comment:
Use `player`  instead of `mc.player` for null safety

Reply:
I have changed this now, thanks